randovania/games/prime1/json_data/Chozo Ruins.txt

```diff
@@ -2239,6 +2257,17 @@ Extra - aabb: [743.8239, -217.36417, 29.29987, 834.11273, -144.96254, 81.37303]
   * Ice Door to Elder Chamber/Door to Hall of the Elders; Dock Lock Rando incompatible with: Missile Blast Shield (randomprime), Super Missile Blast Shield, Power Bomb Blast Shield, Wavebuster Blast Shield, Ice Spreader Blast Shield, Flamethrower Blast Shield
   * Extra - dock_index: 4
   * Extra - nonstandard: False
+  > Door to Reflecting Pool Access
+      All of the following:
+          # https://youtu.be/PAcHvddnSJ0
+          Space Jump Boots and After Elder Chamber Ghost Skip and After Ghost Skip Barrier and Knowledge (Advanced) and Slope Jump (Intermediate)
+  > Door to Crossway Access South
+      All of the following:
+          # Slope Jump: https://www.youtube.com/watch?v=PAcHvddnSJ0 --- Avoid Trigger: https://www.youtube.com/watch?v=C5DBRQ4gCIY
+          Space Jump Boots and Knowledge (Advanced) and L-Jump (Expert) and Slope Jump (Intermediate) and Standable Terrain (Intermediate)
+          Any of the following:
+              After Elder Chamber Ghost Skip
```

Comment:
This is a bad usage of an event and it can come back to haunt us. Randovania cannot support this logic use case right now, period. We have to wait for core randovania to be updated to handle these kinds of edge cases. Please remove.

@henriquegemignani Here's another instance of "there and back" pickup logic edge case. This time instead of a pickup, it's a dead-end room with a pickup inside.

Context: https://discord.com/channels/914291389293027329/945781492360020049/1110375837418651669

Reply:
Also for people outside of that channel, this has some info on it too: https://github.com/randovania/randovania/issues/4456

Reply:
I forgot I made a proper ticket lol

Reply:
only the Ghost Skip event? since the barrier is permanent and I don't think would have any issues?

What makes this application any different than the normal connection though, which is the same but for Statue Moved? Or does that have the same situation